README.md

```diff
@@ -94,6 +95,7 @@ A curated list of tools and resources for Platform Engineering.
 - [OpenCost — open source cost monitoring tool for Kubernetes](https://github.com/opencost/opencost)
 - [Apache SkyWalking — Application Performance Monitoring](https://github.com/apache/skywalking)
 - [SigNoz- an open-source alternative to DataDog, NewRelic, etc.](https://github.com/signoz/signoz)
+- [kube-green- An operator to reduce CO2 footprint of your clusters](https://github.com/kube-green/kube-green)
```

Comment:
As was mentioned in https://github.com/shospodarets/awesome-platform-engineering/pull/3/files#r1116644995
Please remove this link, since it's already in https://github.com/Green-Software-Foundation/awesome-green-software